### PR TITLE
Traditional Chinese | Translation Patches

### PR DESCRIPTION
**Pull request by KiminaNatsuki**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes
Updated the outdated translation.

- [x] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/resources/languages/zh-TW.json [hunk 1]
```diff
@@ -201,7 +211,7 @@
       "description": "殺死所有怪物。"
     },
     "killCharacter": {
-      "usage": "用法：killcharacter [playerId]",
+      "usage": "用法：/killcharacter [playerId]",
```
Comment (Birdulon) [inline src/main/resources/languages/zh-TW.json]: Doesn't seem to match the others

Comment (KiminaNatsuki) [reply src/main/resources/languages/zh-TW.json]: right

File: src/main/resources/languages/zh-TW.json [hunk 2]
```diff
@@ -252,10 +262,13 @@
       "description": "重置當前角色的命之座，重新登入後將會生效。"
     },
     "resetShopLimit": {
-      "usage": "用法：resetshop <playerId>",
+      "usage": "用法：/resetshop <playerId>",
```
Comment (zhaodice) [inline src/main/resources/languages/zh-TW.json]: Doesn't seem to match the others (?

Comment (KiminaNatsuki) [reply src/main/resources/languages/zh-TW.json]: abc

**Review by KingRainbow44 — APPROVED** [review]

**Pull request merged**